### PR TITLE
[docker] disable APM by default, document changes

### PR DESCRIPTION
**Pull request by xvello**
### What does this PR do?

- set `DD_APM_ENABLED` to false by default on the docker image (breaking change)
- hardcode `DD_LOGS_ENABLED=false` and `DD_DOGSTATSD_NON_LOCAL_TRAFFIC=false` in the docker image to be future-proof
- update the documentation

### Motivation

Be consistent with dsd not listening to the network by default (but running for jmxfetch and other agents)

### Additional Notes

This requires RC2 to be built with https://github.com/DataDog/datadog-trace-agent/releases/tag/6.0.0-rc.2

### CODE REVIEW

File: docs/agent/changes.md [hunk 1]
```diff
@@ -193,7 +193,12 @@ The APM agent (also known as _trace agent_) is shipped by default with the
 Agent 6 in the Linux, MacOS and Windows packages.
 
 Similar to the Agent 5, the APM agent is enabled by default. To disable it, set
-`apm_enabled` to `false` in the main agent configuration (`datadog.yaml`).
+`apm_enabled` to `false` in the main agent configuration (`datadog.yaml`). It only
+listens to localhost by default. You can set `apm_config.apm_non_local_traffic = true`
+to enable listening on the network.
+
+In the Docker image, the APM agent is enabled by default. You can enable it by setting
```
Comment (LotharSee) [inline docs/agent/changes.md]: Typo.

> In the Docker image, the APM agent is disabled by default.

**Review by LotharSee — DISMISSED** [review]

File: Dockerfiles/agent/README.md [hunk 2]
```diff
@@ -26,9 +26,11 @@ The agent is highly customizable, here are the most used environment variables:
 
 #### Optional collection agents
 
+These features are disabled by default, you can choose to enable them:
```
Comment (masci) [inline Dockerfiles/agent/README.md]: can we add "for security reasons" or something like that?

File: releasenotes/notes/disable-apm-default-f32eaa7cb3d7f157.yaml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+---
+upgrade:
+  - |
+    Normal installations: APM now listens to localhost only be default, you need to set
```
Comment (masci) [inline releasenotes/notes/disable-apm-default-f32eaa7cb3d7f157.yaml]: typo: s/be/by

**Review by masci — CHANGES_REQUESTED** [review]
left a couple of nits

**Review by masci — DISMISSED** [review]

**Pull request merged**